db/schema.rb

```diff
@@ -73,6 +73,7 @@
     t.integer "location_radius"
```

Comment:
Not necessarily part of this, but I think it would be useful to change this to `location_radius_miles` to reflect the unit.

Reply:
yep